libs/application/templates/aosh/work-accident-notification/src/fields/Components/CausesAndEffects.tsx

```
@@ -0,0 +1,197 @@
+        <Box marginTop={3} border="standard" padding={4}>
+          <Box marginBottom={2}>
+            <AlertMessage
+              type="warning"
+              message={formatMessage(
+                causeAndConsequences.shared.mostSeriousWarning,
+              )}
+            />
+            {errors && getErrorViaPath(errors, answerId.split('.')?.[1] || '') && (
+              <Box paddingTop={2}>
+                <ErrorMessage
+                  children={formatMessage(
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)